README.md

```diff
@@ -3,13 +3,13 @@
 ![](https://travis-ci.org/kurko/ember-json-api.svg?branch=master)
 
 This is a [JSON API](http://jsonapi.org) adapter for [Ember Data](http://github.com/emberjs/data)
-1.0 beta 15, that extends the built-in REST adapter. Please note that Ember Data
+1.11 beta 3, that extends the built-in REST adapter. Please note that Ember Data
```

Comment:
I had fixed it in the bower file, but forgot to update the README. I just pushed that change.